### ISSUE TITLE
Floating save button improvements

### ISSUE THREAD

**Issue by karol-depka** [karol-depka/NetworkingMatcher#156]
1. make the bottom and right margins of the floating save button equal

2. make it save profile (instead of scrolling down).
If privacy checkbox is not checked, then message should appear and viewport should be scrolled to the checkbox; as well as the checkbox should flash for a moment to highlight it

RACI:
Responsible: @zolastro 
Accountable: @AlejandroMornie 
Consulted: @karol-depka

**Comment by karol-depka** [comment]
@Alkesst said it is mind blowing that the button scrolls down ;)

**Comment by Alkesst** [comment]
Yes, the button on the cloud has an arrow pointing up and it could be confusing :) In fact, I thought it was a button to scroll up the page ;)

**Comment by zolastro** [comment]
I've modified the icon and the behaviour of the button. Also corrected the margins. Still have to animate the privacy checkbox ;)

**Comment by zolastro** [comment]
After the new sidenav changes, the save button no longer scrolls down. This is due to the fact that now is not the body the element that should scroll down, but the sidenav content component.
